nocodb/docker-compose.yml

```
@@ -0,0 +1,35 @@
+version: "3.7"
+
+services:
+
+  nocodb:
+    image: nocodb/nocodb:0.265.1@sha256:8fd57018accf775fb5390392910e4b0ef6bb154f17c5732b478eb76c40ee8d1e
+    restart: on-failure
+    user: "1000:1000"
+    depends_on:
+      postgres:
+        condition: service_healthy
+    ports:
+      - 2183:8080
+    environment:
+      NC_DB: "pg://nocodb_postgres_1:5432?u=nocodb&p=itworkslocally&d=nocodb"
+    volumes:
+      - ${APP_DATA_DIR}/data/nc_data:/usr/app/data
```

Rename folder to `app`.
```suggestion
      - ${APP_DATA_DIR}/data/app:/usr/app/data
```